sdks/paths.mk

```diff
@@ -43,7 +43,7 @@ ifndef DISABLE_IOS
 CheckXcodeDir=$(or $(and $(wildcard $(1))),$(warning Could not find Xcode in "$(1)"))
 
 ifneq ($(call CheckXcodeDir,/Applications/Xcode.app/Contents/Developer),)
-XCODE_DIR=/Applications/$(XCODE_VERSION).app/Contents/Developer
+XCODE_DIR=$(XCODE_DEVELOPER_ROOT)
```

Comment:
the `CheckXcodeDir` if above should be changed to check `$(XCODE_DEVELOPER_ROOT)`  too